components/sections/NavigationBar/index.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import NavigationBar from './index';
```

Comment:
```suggestion
import NavigationBar from '@/components/sections/NavigationBar';
```

Need to use absolute path.

@ovflowd I think the location of the sections has been updated to `@/components/Containers`, shouldn't we move this section there as well?

Reply:
Yes, NavigationBar should be moved to the "Containers" folder